lib/std/os/linux.zig

```diff
@@ -109,36 +109,136 @@ pub const SYS = switch (@import("builtin").cpu.arch) {
     else => @compileError("The Zig Standard Library is missing syscall definitions for the target CPU architecture"),
 };
 
-pub const MAP = struct {
-    pub usingnamespace arch_bits.MAP;
-
-    /// Share changes
```

Comment:
This dropped the doc-comments.
Is there a nice way to keep them?

Reply:
There is, but they don't look too valuable to me.